src/routes/ViewAllLogs/ViewAllLogs.js

```diff
@@ -411,13 +408,19 @@ class ViewAllLogs extends Component {
     } = this.props;
     const { isLogsDeletionInProgress } = this.state;
 
+    let isAllSelectedByValues = isAllSelected;
+    if (isAllSelected) {
+      const nonEmptyRecords = resources.records.records.map(record => record.id).filter(Boolean);
```

Comment:
Could you change this part in manifest something like this:
```suggestion
      const nonEmptyRecords = resources.jobLogs.records.map(record => record.id).filter(Boolean);
```

Reply:
already done with jobExecutions.